src/dataflow/components/dataflow-program.tsx

```diff
@@ -267,6 +267,13 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
 
       const program = this.props.program && safeJsonParse(this.props.program);
       if (program) {
+        if (this.getRunState() === ProgramRunStates.Complete) {
+          Object.keys(program.nodes).forEach(id => {
```

Comment:
1. Consider moving this to a named helper function or at least adding a comment.
2. Consider...
  a. adding a local variable `const node = program.nodes[id];` to reduce redundancy and enhance clarity.
  b. alternatively, consider lodash `forEach` which could look something like this:
```
forEach(program.nodes, node => node.data?.plot && node.data.plot = false);
```

Reply:
Agreed.  Made a function and added lodash `forEach`